front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings/index.tsx

```diff
@@ -9,11 +9,20 @@ import {
 import CTASignedOutSettings from './CTASignedOutSettings';
 import CTASignedInSettings from './CTASignedInSettings';
 import { CLErrors } from 'typings';
+import { IHomepageSettings } from 'services/homepageSettings';
+import 'utils/moduleUtils';
 
-interface Props {
+declare module 'utils/moduleUtils' {
```

Comment:
Came to the realization that we can much more easily add these via the modules, so we don't need to redefine the props in several places.

[Full commit](https://github.com/CitizenLabDotCo/citizenlab/pull/2303/commits/c90376408ca213537ccb9313e03917fe30f5c485)

Reply:
Will mention in #chapter-front-end/CDM once this iteration is over.